macros/utils/required_docs/evaluate_required_docs.sql

```diff
@@ -28,7 +28,9 @@
 
             {% for column in model_columns %}
 
-                {% set column = column | lower %}
+                {% if var("convert_column_names_to_lower_case", true) is true %}
```

Comment:
nit, extra bool comparison 
```suggestion
                {% if var("convert_column_names_to_lower_case", true) %}
```